### ISSUE TITLE
Support multiple linters per file/language

### ISSUE THREAD

**Issue by Mark-Simulacrum** [MiguelCastillo/Brackets-InteractiveLinter#78]
LinterManager should allow more than one linter to be registered per language, and should loop through all linters registered for a language and report errors. I am ready to work on the UI part of this implementation (making the inline widgets and panel support multiple linters) - but I am not sure how to do the LinterManager part (I have not worked much with the plugins or workers).

This is the remainder of #33, so I thought we should move it into its own issue.

This is the main thing I'm waiting on before work on #52.

**Comment by cgfrost** [comment]
I'd like to bump this one, I'm trying to use JSCS and JSHint together. I'm very happy to do testing from a branch if it helps.

I assume the config in `brackets.json` should look like this once implemented.
```
"interactive-linter.javascript": ["jshint", "jscs"]
```
